README.md

```diff
@@ -56,11 +56,9 @@ Docker should function normally, with the following caveats:
 * [Setup a secure private registry](registry-example.md)
 
 
-## NVIDIA support on Ubuntu Core 22
+## NVIDIA support
```

Comment:
Please, add a note explaining the binary discovery path problem for `nvidia-smi`.

Reply:
I've attempted to clarify the usage notes more generally, and also included a short section on that.

Please let me know if that section of the README looks ok, or it needs work still.

Reply:
I've checked it, and it's perfect! Thank you very much!